spec/api/sul_bib/authorship_api_spec.rb

```diff
@@ -437,7 +385,7 @@ def check_response_error(code)
       end
 
       context 'with cap_profile_id' do
-        let(:author_hash) { cap_author_hash }
+        let(:author_hash) { Hash[cap_profile_id: author.cap_profile_id] }
         it_behaves_like 'it creates new contributions and publications'
```

Comment:
Ah, here it is.  So `cap_author_hash` has disappeared.  OK, if this is the only place it was used.